src/components/TaskItem.tsx

```diff
@@ -1,23 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 interface TaskItemProps {
   taskName: string;
+  deleteTask: () => void;
 }
 
-export default function TaskItem({ taskName }: TaskItemProps) {
+export default function TaskItem({ taskName, deleteTask }: TaskItemProps) {
+  const [edit, setEdit] = useState<boolean>(false);
```

Comment:
É bobagem, mas o nome correto dessa variavel seria `isEditing` (esta editando)
```suggestion
  const [isEditing, setIsEditing] = useState<boolean>(false);
```